src/plugins/kibana/server/routes/api/ingest/register_pipelines.js

```diff
@@ -0,0 +1,54 @@
+import _ from 'lodash';
+import esToKibanaPipelineConverter from '../../../lib/converters/ingest_pipeline_api_es_to_kibana_converter';
+import handleESError from '../../../lib/handle_es_error';
+
+function handlePipelinesResponse(response) {
+  const pipelines = _.get(response, 'pipelines');
+  const results = _.map(pipelines, 'id');
```

Comment:
What's the reason for returning only the id? Data over the wire shouldn't be much of an issue. If we return the full pipelines here, the additional round trip after the user selects the pipeline can be avoided. 

If there is a compelling reason to return only the ID, I'd like to make that filtering more explicit by requiring the client to add something like a `fields` query parameter specifying the fields they want (in this case, just id). That way the client can decide how much or little info they want.